apps/nuxt3-ssr/components/layouts/DetailPage.vue

```diff
@@ -33,5 +33,9 @@ const bannerHtml = computed(() => {
       </div>
     </div>
   </Container>
-  <FooterComponent />
+  <FooterComponent>
+    <ClientOnly>
```

Comment:
why ClientOnly? Will maybe remove this comment if I see why.

In practice we should always see this except in 'dev' mode?